covert/__main__.py

```diff
@@ -109,11 +110,20 @@ def argparse():
   elif next((k for k, v in encargs.items() if av[0] in v), None) is not None:
     args.mode, ad, modehelp = 'enc', encargs, enchelp
     av.insert(0, None)
+  elif av[0].startswith('-'):
+    if av[0][:2] == '-e':
+      args.mode, ad, modehelp = 'enc', encargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
+    elif av[0][:2] == '-d':
+      args.mode, ad, modehelp = 'dec', decargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
```

Comment:
This works but is there any chance that you could refactor this together with the previous ifs on lines 104-107? We would prefer not having to repeat lines 115 and 117 here. Use `or av[0].startswith("-e")` in the condition of the original if.